packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -34,6 +34,11 @@ const RouteWithSubRoutes = observer(route => {
             return false;
         } else if (pathname.includes(routes.cashier_v2) && !is_next_cashier_enabled) {
             return false;
+        } else if (
+            (pathname === routes.traders_hub_v2 || pathname === routes.compare_accounts) &&
```

Comment:
```suggestion
            pathname.includes(routes.traders_hub_v2) &&
```

Reply:
@mahdiyeh-deriv I agree with this however this cannot be used as this condition is true with `/appstore/traders-hub` as well. Initially we implemented it like this.

Reply:
I see, thanks for clarification